x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/service_inventory/service_inventory.spec.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import url from 'url';
+import { synthtrace } from '../../../../synthtrace';
+import { opbeans } from '../../../fixtures/synthtrace/opbeans';
+
+const timeRange = {
+  rangeFrom: '2021-10-10T00:00:00.000Z',
+  rangeTo: '2021-10-10T00:15:00.000Z',
+};
+
+const serviceInventoryHref = url.format({
+  pathname: '/app/apm/services',
+  query: timeRange,
+});
```

Comment:
Is it possible to use `apmRouter` to produce links?
```ts
const serviceInventoryHref = apmRouter.link('/services', { query: { timeRange }});
```

Reply:
It fails when I import something from '/public'

Reply:
Ah, okay!

Reply:
Is it just an eslint warning or does it actually fail compilation? If it's just a warning I think we can suppress it.

Reply:
... that being said, don't let this block you.

Reply:
It fails compilation, I could investigate